robotoff/workers/update_listener.py

```diff
@@ -102,16 +106,43 @@ def process_redis_update(self, redis_update: RedisUpdate):
                 )
 
 
+@backoff.on_exception(
+    backoff.expo,
+    ConnectionError,
+    max_tries=10,
+    max_time=300,  # 5 minutes
```

Comment:
I would suggest to:
- remove `max_tries` and `max_time` completely. We get all our events from Redis, we want to try again until Redis is not down.
- Limit the maximum time to wait to 60s using `max_value`: it's not a web server with a public API, it's okay to give it a try every minute. 

```suggestion
    max_value=60,  # we wait at most 60 seconds between retries
```